v2/api/kubernetesconfiguration/v1api20230501/structure.txt

```diff
@@ -255,3 +255,420 @@ Extension_Spec_ARM: Object (5 properties)
         ├── "Key"
         ├── "ManagedIdentity"
         └── "User"
+FluxConfiguration: Resource
+├── Spec: Object (14 properties)
+│   ├── AzureBlob: *Object (9 properties)
+│   │   ├── AccountKey: *genruntime.SecretReference
+│   │   ├── ContainerName: *string
+│   │   ├── LocalAuthRef: *string
+│   │   ├── ManagedIdentity: *Object (1 property)
+│   │   │   └── ClientId: *string
+│   │   ├── SasToken: *genruntime.SecretReference
+│   │   ├── ServicePrincipal: *Object (8 properties)
+│   │   │   ├── ClientCertificate: *genruntime.SecretReference
+│   │   │   ├── ClientCertificatePassword: *genruntime.SecretReference
+│   │   │   ├── ClientCertificateSendChain: *bool
+│   │   │   ├── ClientId: *string
+│   │   │   ├── ClientIdFromConfig: *genruntime.ConfigMapReference
+│   │   │   ├── ClientSecret: *genruntime.SecretReference
+│   │   │   ├── TenantId: *string
+│   │   │   └── TenantIdFromConfig: *genruntime.ConfigMapReference
+│   │   ├── SyncIntervalInSeconds: *int
+│   │   ├── TimeoutInSeconds: *int
+│   │   └── Url: *string
+│   ├── AzureName: string
+│   ├── Bucket: *Object (7 properties)
+│   │   ├── AccessKey: *genruntime.SecretReference
+│   │   ├── BucketName: *string
+│   │   ├── Insecure: *bool
+│   │   ├── LocalAuthRef: *string
+│   │   ├── SyncIntervalInSeconds: *int
+│   │   ├── TimeoutInSeconds: *int
+│   │   └── Url: *string
+│   ├── ConfigurationProtectedSettings: *genruntime.SecretMapReference
+│   ├── GitRepository: *Object (8 properties)
+│   │   ├── HttpsCACert: *genruntime.SecretReference
+│   │   ├── HttpsUser: *string
+│   │   ├── LocalAuthRef: *string
+│   │   ├── RepositoryRef: *Object (4 properties)
+│   │   │   ├── Branch: *string
+│   │   │   ├── Commit: *string
+│   │   │   ├── Semver: *string
+│   │   │   └── Tag: *string
+│   │   ├── SshKnownHosts: *string
+│   │   ├── SyncIntervalInSeconds: *int
+│   │   ├── TimeoutInSeconds: *int
+│   │   └── Url: *string
+│   ├── Kustomizations: map[string]Object (9 properties)
+│   │   ├── DependsOn: string[]
+│   │   ├── Force: *bool
+│   │   ├── Path: *string
+│   │   ├── PostBuild: *Object (2 properties)
+│   │   │   ├── Substitute: map[string]string
+│   │   │   └── SubstituteFrom: Object (3 properties)[]
+│   │   │       ├── Kind: *string
+│   │   │       ├── Name: *string
+│   │   │       └── Optional: *bool
+│   │   ├── Prune: *bool
+│   │   ├── RetryIntervalInSeconds: *int
+│   │   ├── SyncIntervalInSeconds: *int
+│   │   ├── TimeoutInSeconds: *int
+│   │   └── Wait: *bool
+│   ├── Namespace: *string
+│   ├── Owner: *genruntime.ArbitraryOwnerReference
+│   ├── ReconciliationWaitDuration: *string
+│   ├── Scope: *Enum (2 values)
+│   │   ├── "cluster"
+│   │   └── "namespace"
+│   ├── SourceKind: *Enum (3 values)
+│   │   ├── "AzureBlob"
+│   │   ├── "Bucket"
+│   │   └── "GitRepository"
+│   ├── Suspend: *bool
+│   ├── SystemData: *Object (6 properties)
```

Comment:
I think that systemdata is expected to be readonly? That's what https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v6/types.json#L31 says at least.

Reply:
I think we can just prune this from the spec.

Reply:
Done!